src/shared/components/table/Table.tsx

```diff
@@ -0,0 +1,51 @@
+import csx from './Table.scss';
```

Comment:
Move csx import after react-window and add enter between table namespace and FixedSizeList